src/shared/particle_dynamics/solid_dynamics/thin_structure_dynamics.cpp

```diff
@@ -101,7 +101,7 @@ ShellStressRelaxationFirstHalf::
         gaussian_weight_ = three_gaussian_weights_;
     }
     /** Define the factor of hourglass control algorithm. */
-    hourglass_control_factor_ = 0.002;
+    hourglass_control_factor_ = 0.01;
```

Comment:
I remember Dong once said that increasing the hourglass factor could also reduce the accuracy. Maybe it would be better to make this factor an input parameter instead of changing it in a hardcoded way

Reply:
Bubbling it up means adding another black box knob that users might get lost into. Might be worth it or not, I do not have opinion on that. The main question then is more: modifying here or adding a duplicate class in our codebase ?

BTW please comment on the PR (#126 and #125) instead of here. This one is only the wip I use for the medtronic test run